notebooks/nzh-20230417-00-embeddings-classifier.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.            precision_score(truth, preds, average="macro"),
</pre><p>I think you made some copy paste errors here, the recall and f1_score lines are averaging over the precision.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>